src/u_cx_at_client.c

```diff
@@ -483,7 +483,8 @@ int32_t uCxAtClientExecSimpleCmdF(uCxAtClient_t *pClient, const char *pCmd, cons
 
 int32_t uCxAtClientExecSimpleCmd(uCxAtClient_t *pClient, const char *pCmd)
 {
-    cmdBeginF(pClient, pCmd, "", NULL);
+    va_list args;
+    cmdBeginF(pClient, pCmd, "", args);
```

Comment:
Ahh.. yes, this was bug. However, we want a `NULL` in the `va_list` and right now it will be uninitialized. But I think this should do the trick:
```c
int32_t uCxAtClientExecSimpleCmd(uCxAtClient_t *pClient, const char *pCmd)
{
   return uCxAtClientExecSimpleCmdF(pClient, pCmd, "", NULL);
}
```